### ISSUE TITLE
How to compile program with debug symbols information?

### ISSUE THREAD

**Issue by AlexRadch** [RustDT/RustDT#88]
Can you describe how to  compile program with debug symbols information here https://github.com/RustDT/RustDT/blob/latest/documentation/UserGuide.md
Not only note about it!

> **Note that for debugging to work**, the program must be compiled with debug symbols information, and those debug symbols must be on a format that GDB understands. Otherwise you will get GDB error messages such "(no debugging symbols found)" or "file format not recognized".

**Comment by bruno-medeiros** [comment]
By default, when `cargo build` is used to build, the executables will have debugging information. Only if you use the `--release` option are debugging symbols stripped away.

**Comment by AlexRadch** [comment]
Why I got next error `No source available for "main() at 0x4015ca"` when I try to debug program?
Debugger can not find source file!

![eclipse_2016-02-18_01-05-21](https://cloud.githubusercontent.com/assets/6407980/13121259/d3c1e5dc-d5db-11e5-9927-27ef985c8f74.png)

I use Windows and I configured RustDT by [User Guide](https://github.com/RustDT/RustDT/blob/latest/documentation/UserGuide.md).

After build I have only one file `hello_world.exe` in `target\debug` folder and some small files in `target\debug\.fingerprint` folder.

For example `dep-bin-hello_world` file have next content

```
C:\Users\Alex\Documents\Home\Rust\hello_world

C:\Users\Alex\Documents\Home\Rust\hello_world\target\debug\hello_world.d: src\main.rs

src\main.rs:
```

How to configure Eclipse debugger on Windows to find Rust sources?
